### ISSUE TITLE
Add Custom color and define the color for bundle light and dark (question)

### ISSUE THREAD

**Issue by iruizeguilaz** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3608]
### Bug explanation

question (not bug)

I have and app wpf net7.0 with  MaterialDesignThemes" Version="5.0.0" 

I allow users to choose beetwen light and dark basetheme and I am using primary color BlueGrey and second color Lime

  <materialDesign:BundledTheme BaseTheme="Light" PrimaryColor="BlueGrey" SecondaryColor="Lime" />
ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesign3.Defaults.xaml" 

I have a title and I would like to put a red color when I am in ligth theme but in another color, For instance, white when the app changes into dark theme

It is posible to add cusctom colors linked to the basetheme? 

### Version

package 5.0,0

**Comment by Keboo** [comment]
Yes it is certainly possible, but for the color switching you will likely end up in C# to make the needed changes. Check out the advanced theming on the wiki here: https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Advanced-Theming
